features/.nav

```diff
@@ -44,22 +44,21 @@
   - color.feature
   - fail_fast.feature
   - custom_settings.feature
-  - alias_example_to.feature
   - default_path.feature
 - expectation_framework_integration:
-  - configure_expectation_framework.feature
   - failure_aggregation.feature
-- mock_framework_integration:
-  - use_rspec.feature
-  - use_flexmock.feature
-  - use_mocha.feature
-  - use_rr.feature
-  - use_any_framework.feature
 - formatters:
   - text_formatter.feature
   - configurable_colors.feature
   - json_formatter.feature
   - custom_formatter.feature
-- spec_files:
-  - arbitrary_file_suffix.feature
 - core_standalone.feature
+- using_alternate_tools_and_libraries:
+  - use_flexmock.feature
+  - use_mocha.feature
+  - use_rr.feature
+  - use_any_framework.feature
+  - configure_expectation_framework.feature
+- legacy_features_and_workarounds:
+  - arbitrary_file_suffix.feature
+  - alias_example_to.feature
```

Comment:
The nice thing about `tbd` as a category is it ensures we don't forget to address those, kinda like variables you temporarily name `fixme` or whatever.